packages/radix-ui-themes/src/components/link.css

```diff
@@ -17,7 +17,7 @@
     outline-offset: 2px;
   }
 
-  :where([data-accent-color]:not(.radix-themes, .rt-high-contrast)) & {
+  :where([data-accent-color]:not(.radix-themes, .rt-high-contrast)) &:where([data-accent-color='']) {
```

Comment:
This is the actual Link fix

Reply:
When is it `''`?

Reply:
`data-accent-color` is an empty string whenever no color is passed and it uses the current accent (via Text)

What we want is auto high contrast when you are not setting an explicit `color` prop, which we broke at some point

Reply:
Yeah saw that in the other PR, thanks!